manifests/deployment.yml

```diff
@@ -29,7 +29,7 @@ spec:
       serviceAccount: karydia
       containers:
       - name: karydia
-        image: karydia/karydia:0.1.0
+        image: eu.gcr.io/gardener-project/karydia/karydia
```

Comment:
I think we should still specify a concrete image version (tag) for production deployments. This would provide an easy way to update or to stay on a specific karydia version. Futhermore, it would be easier to identify which karydia version runs on a cluster.

Reply:
Current version is 0.0.1 right?

Reply:
Hmm... that's probably an issue. I think with our new (& current) CI/CD pipeline the versioning isn't in a final state at the moment.
@dacappo, we should discuss this point.